src/test/scala/cornichonTests/authTest.scala

```diff
@@ -5,9 +5,11 @@ class AuthTest extends CornichonFeature with FeatureConfig {
   def feature = Feature("checking the API request"){
     Scenario("API request returns 200"){
 
-      WithBasicAuth (configuration.clientId,configuration.clientSecret){
-        When I post(s"$authUrl")
-        .withParams("grant_type" -> "client_credentials", "scope" -> s"manage_project:$projectKey")
+      WithBasicAuth (clientId,clientSecret){
```

Comment:
a common style in scala:
```suggestion
      WithBasicAuth(clientId, clientSecret) {
```

Reply:
done